components/service/glean/docs/pings.md

```diff
@@ -0,0 +1,46 @@
+# glean pings
+
+If data collection is enabled, glean provides a set of pings that are assembled, out of the box,
```

Comment:
nit:  As a native English speaker the commas around `out of the box` are a little awkward here.  It should still be grammatically correct to say:

`If data collection is enabled, glean provides a set of pings that are assembled out of the box without any developer intervention.`